cmd/subcommands/keys.go

```diff
@@ -91,10 +91,30 @@ func keysSub() []*cobra.Command {
 		},
 	}
 	importP := `passphrase of key being imported, default assumes ""`
-	cmdImport.Flags().StringVar(&importPassphrase, "passphrase", "", importP)
-	cmdImport.Flags().BoolVar(&quietImport, "quiet", false, "do not print out imported account name")
+	cmdImportKS.Flags().StringVar(&importPassphrase, "passphrase", "", importP)
+	cmdImportKS.Flags().BoolVar(&quietImport, "quiet", false, "do not print out imported account name")
+	cmdImportSK := &cobra.Command{
+		Use:   "import-private-key <secp256k1_PRIVATE_KEY> [ACCOUNT_NAME]",
+		Short: "Import an existing keystore key (only accept secp256k1 private keys)",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			passphrase := c.DefaultPassphrase
+			if userProvidesPassphrase {
+				passphrase = doubleTakePhrase()
+			}
+			userName := ""
+			if len(args) == 2 {
```

Comment:
Looking into letting `cobra` handle some of this logic for you/enforce, see 		`Args:  cobra.ExactArgs(1),` in `cmd/subcommands/balance.go:46`

Reply:
added arg range check